customers/tests/test_customers_mutations.py

```diff
@@ -38,48 +48,66 @@
 @pytest.mark.parametrize(
     "api_client", ["berth_services"], indirect=True,
 )
-@freeze_time("2020-01-01T08:00:00Z")
-def test_create_boat_certificate(api_client, boat):
-    boat_id = to_global_id(BoatNode, boat.id)
-    file_name = "certificate_file.pdf"
-    check_date = today()
+def test_create_boat(api_client, customer_profile, boat_type):
+    check_date = today().date()
+    file_name = "certificate.pdf"
+    owner_id = to_global_id(ProfileNode, customer_profile.id)
+    boat_type_id = str(boat_type.id)
 
     variables = {
-        "boatId": boat_id,
-        "file": SimpleUploadedFile(
-            name=file_name, content=None, content_type="application/pdf"
-        ),
-        "certificateType": random.choice(list(BoatCertificateType)).name,
-        "validUntil": check_date,
-        "checkedAt": check_date,
-        "checkedBy": "John Wick",
+        "ownerId": owner_id,
+        "boatTypeId": boat_type_id,
+        "name": "Flying Dutchman",
+        "length": Decimal(random.randrange(1, 99)),
```

Comment:
we do `round(Decimal(random.randrange(1, 99)), 1)` or `round(Decimal(random.randrange(1, 99)), 2)` in other places 🤔 should we be more consistent 🤔 should we wrap this into a util 🤔

Reply:
True, would be nicer to have a single util for all this. There's `payments.utils.round_price`, but definitely worth extracting. Good catch. 👍